src/user/apps/discourse/discourse-components/category-details.js

```diff
@@ -0,0 +1,107 @@
+import React, {Component} from 'react';
+import Navigation from '../../../dashboard/navigation/navigation';
+import {Table} from 'react-bootstrap';
+import Spinner from 'react-bootstrap/Spinner'
+import './category-details.scss'
+import {Link} from 'react-router-dom'
+
+
+const organizationUrl = 'https://meta.discourse.org'
+
+class CategoryDetails extends Component{
+
+    constructor(props){
+        super(props);
+        this.state={
+            loading: true,
+            topics:[],
+            category: '',
+            discourse: true
+        }
+    }
+
+    async componentDidMount(){
+        await fetch(`https://cors-anywhere.herokuapp.com/${organizationUrl}/c/${this.props.match.params.id}`, {
```

Comment:
.

Reply:
@harsh253 can you tell what is this actually?

Reply:
So basically this is a very basic step towards the integration of Discourse in the portal.  When we go to https://deploy-preview-366--donut-frontend-r.netlify.com/apps/discourse, we can see 2 cards.
One shows a list of all categories and one shows list of latest topics from the discourse channel
Here https://github.com/codeuino/social-platform-donut-frontend/pull/366/files/989d223c51d92e968acda49b7d95e4e8bf12bae1#diff-e4b44e5c75fb9aa3cc68c0660f9d42c4 , I am calling the fetch function to make an API call to the mentioned route to get all the list of topics under a particular category. 
So as soon as you select a category from the 1st card from the left , we get a list of topics under that category using this function.